cleanlab_studio/studio/trustworthy_language_model.py

```diff
@@ -406,7 +431,12 @@ def try_get_trustworthiness_score(
         """
         validate_try_tlm_prompt_response(prompt, response)
 
-        return self._batch_get_trustworthiness_score_with_handled_errors(prompt, response)
+        return cast(
+            List[Optional[float]],
```

Comment:
[nit] should return type be `TryBatchGetTrustworthinessScoreResponse`? Same with a bunch of `get_trustworthines_score` -related methods